lib/matplotlib/colorbar.py

```diff
@@ -1165,24 +1165,36 @@ def make_axes(parents, location=None, orientation=None, fraction=0.15,
     return cax, kw
 
 
+# helper functions for row,col to index.
+def index2rowcolunm(index, ncols):
+    col = index%ncols + 1
+    row = int(np.floor(index/ncols)+1)
+    return row, col
+
+
+def rowcolunm2index(row, col, ncols):
+    index = col-1+(row-1)*ncols
+    return index
+
+
 @docstring.Substitution(make_axes_kw_doc)
-def make_axes_gridspec(parent, **kw):
+def make_axes_gridspec(parents, **kw):
```

Comment:
Changing the argument name is an API change (as `make_akes_gridspec(parent=obj)` will be broken).